test/testDelete.js

```diff
@@ -0,0 +1,125 @@
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('ok').and.to.be.eql(1);
+                        res.body.should.have.property('n').and.to.be.eql(1);
+                        res.body.should.have.property('deletedCount').and.to.be.eql(1);
+                        done();
+                    })
+            })
+        })
+
+        describe("Three issues", () => {
+            it("status code 200, issues deleted", (done) => {
+                var n = 3;
+                for (var i = 0; i < n; i++) addIssue();
+                chai.request(app)
+                    .delete('/api/issues/delete/all')
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('ok').and.to.be.eql(1);
+                        res.body.should.have.property('n').and.to.be.eql(n);
+                        res.body.should.have.property('deletedCount').and.to.be.eql(n);
+                        done();
+                    })
+            })
+        })
+
+        describe("Delete all but no issues in db", () => {
+            it('status code 200, no issues deleted', (done) => {
+                chai.request(app)
+                    .delete('/api/issues/delete/all')
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('ok').and.to.be.eql(1);
+                        res.body.should.have.property('n').and.to.be.eql(0);
+                        res.body.should.have.property('deletedCount').and.to.be.eql(0);
+                        done();
+                    })
+            })
+        })
+    })
+
+    describe("DELETE /api/issues/:id", () => {
+        describe("One issue", () => {
+            it('status code 200, issue deleted', (done) => {
+                var n = 1;
+                var result = addIssue();
+                chai.request(app)
+                    .delete('/api/issues/' + result._id)
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('deleted').and.to.be.an('object');
+                        var deleted = res.body.deleted;
+                        deleted.should.have.property('title').and.to.be.eql(result.title);
+                        deleted.should.have.property('user').and.to.be.eql(result.user);
+                        deleted.should.have.property('text').and.to.be.eql(result.text);
+                        deleted.should.have.property('closed').and.to.be.eql(result.closed);
+                        done();
```

Comment:
Same as above, your test checks if response is ok (which is fine) but it doesn't mean that issue is deleted for sure